sdk/iOS/src/MSSyncTable.h

```diff
@@ -96,12 +97,19 @@
 /// @{
 
 /// Initiates a request to go to the server and get a set of records matching the specified
-/// MSQeury object.
+/// MSQuery object.
 /// Before a pull is allowed to run, one operation to send all pending requests on the
 /// specified table will be sent to the server. If a pending request for this table fails,
 /// the pull will be cancelled
 -(NSOperation *)pullWithQuery:(MSQuery *)query queryId:(NSString *)queryId completion:(MSSyncBlock)completion;
 
+/// Initiates a request to go to the server and get a set of records matching the specified
+/// MSQuery object.
+/// Before a pull is allowed to run, one operation to send all pending requests on the
+/// specified table will be sent to the server. If a pending request for this table fails,
+/// the pull will be cancelled
+-(NSOperation *)pullWithQuery:(MSQuery *)query queryId:(NSString *)queryId pullSettings:(MSPullSettings *)pullSettings completion:(MSSyncBlock)completion;
```

Comment:
Let's add the annotations in the MobileApp branch. Partially annotating a header file results in warnings. I don't want to annotate the entire header file in dev branch to avoid potential merge conflicts.